.github/workflows/link-validator.yml

```diff
@@ -1,13 +1,13 @@
-name: Check links for modified files
+name: Check links in Markdown files
 
-on:
+"on":
```

Comment:
Because of yamllint.

```
❯ yamllint x.yml
x.yml
  2:1       warning  truthy value should be one of [false, true]  (truthy)

❯ cat x.yml
---
on:
  - foo
 ```

```
❯ yamllint x.yml
❯ cat x.yml
---
"on":
  - foo
```